tim1/src/main/java/isamrs/tim1/service/RegisteredUserService.java

```diff
@@ -31,7 +31,7 @@
 	@Autowired
 	private RegisteredUserRepository registeredUserRepository;
 
-	public ArrayList<UserDTO> searchUsers(String firstName, String lastName, String email) {
+	public ArrayList<UserDTO> searchUsers(String firstName, String lastName) {
 		if (firstName.equals(""))
```

Comment:
Change this to firstName = firstName.toLowerCase() + "%"

Reply:
Suggestion is accepted, it will be corrected in next request